### PR TITLE
fix(ui): paginator `klass` option rename and became optional

### PR DESCRIPTION
**Pull request by ksnitsky**
- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

**Review by wrmk — APPROVED** [review]

**Comment by wrmk** [comment]
btw, do we want to remove `mt-3` in activities tab for candidates?

**Comment by ksnitsky** [comment]
> btw, do we want to remove `mt-3` in activities tab for candidates?

I moved it to a paginator base class list

File: app/views/kaminari/ats/_paginator.html.slim [hunk 1]
```diff
@@ -5,9 +5,9 @@
     per_page:      number of items to fetch per page
     remote:        data-remote
     paginator:     the paginator that renders the pagination tags inside
-    klass:         additional classes for wrapper
+    class:         additional classes for wrapper
 == paginator.render do
-  nav.page-paginator aria-label="pager" class=klass
+  nav.page-paginator.mt-3 aria-label="pager" class=local_assigns[:class]
```
Comment (ksnitsky) [inline app/views/kaminari/ats/_paginator.html.slim:10]: Here

Comment (wrmk) [reply app/views/kaminari/ats/_paginator.html.slim:10]: i mean not only activities for positions, but also for candidates
it is different files

Comment (ksnitsky) [reply app/views/kaminari/ats/_paginator.html.slim:10]: This file is used by all paginators

Comment (wrmk) [reply app/views/kaminari/ats/_paginator.html.slim:10]: `/ats/candidates/activities_tab.html.slim` also contains `mt-3`. will be removed in separate pr
agreed on a call.

**Pull request merged**